bezier_library/include/bezier_library/bezier_library.hpp

```diff
@@ -49,6 +53,18 @@ typedef pcl::PointXYZRGBA PointT;
 /** @brief PointCloudT is a PointT cloud */
 typedef pcl::PointCloud<PointT> PointCloudT;
 
+/** @brief Display is an enum serving as parameters in display functions
+ *  @enum GRINDING @brief display items related to the grinding path
+ *  @enum EXTRICATION @brief display items related to the extrication path
+ *  @enum ALL @brief display items related both to the grinding and extrication path
+ *  */
```

Comment:
This is how enums should be documented with Doxygen:
https://github.com/PointCloudLibrary/pcl/pull/1581/files#diff-aa3884f304d05ba2c859586230878e3aL102